ner_v2/detectors/numeral/utils.py

```diff
@@ -1,7 +1,9 @@
 from __future__ import absolute_import
+
 import re
 from six.moves import range
 
+
 def get_number_from_number_word(text, number_word_dict):
```

Comment:
I haven't clearly understood how the number_word_dict's scale and increment help in reconstructing the number here. Can you describe the functionality a little bit or let me know if you want to get on-call for this. I will re-review the double, and triple functionality after understanding the flow again.

Reply:
Let's get on a call.
Based on which pointers are not clear, I will add relevant explanatory comments

Reply:
Hey, my fundamental and only doubt here is what scale and increment mean in the number_word_dict.